src/interpreter_funcs/separate_isolated_meshes.rs

```diff
@@ -28,19 +28,15 @@ impl Func for FuncSeparateIsolatedMeshes {
     }
 
     fn return_ty(&self) -> Ty {
-        Ty::Geometry
+        Ty::GeometryArray
     }
 
     fn call(&mut self, args: &[Value]) -> Result<Value, FuncError> {
         let geometry = args[0].unwrap_geometry();
 
-        let values = mesh_tools::separate_isolated_meshes(&geometry);
+        let geometries = mesh_tools::separate_isolated_meshes(&geometry);
```

Comment:
off topic: we should consider renaming this to `disjoint_mesh` during the geometry cleanup

Reply:
I think I've seen a new trello issue for that. Please prioritize it in the backlog accordingly.